less/pages/buyers-guide/shared.less

```diff
@@ -97,7 +101,7 @@
       }
 
       &.home-link a {
-        background-image: url(/assets/buyers-guide/thumbnails/home-thumbnail.jpg);
+        background-image: url(/assets/buyers-guide/thumbnails/01-toys-thumbnail.jpg);
```

Comment:
Can you revert these changes here?

I think this is due to changes I made after you.